gh_notifs.py

```diff
@@ -275,8 +275,9 @@ def _icons(notif: Notification) -> Iterator[str]:
     def _target_branch(pr: PR) -> str:
         if pr.base_ref != pr.base_default_branch:
             return f"""\
+<br/>
```

Comment:
I think this should go above line 317 in the overall notification template, it's part of the overall layout, not the rendering of the target branch specifically.

Also this means we're putting a linebreak inside a span, which is a bit unexpected.

Reply:
Good call. I've amended that final commit to move the BR now.